app/src/main/java/com/google/android/gms/samples/wallet/CheckoutActivity.java

```diff
@@ -118,7 +117,9 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                         break;
                     case AutoResolveHelper.RESULT_ERROR:
                         Status status = AutoResolveHelper.getStatusFromIntent(data);
-                        handleError(status.getStatusCode());
+                        if (status != null) {
```

Comment:
This is a bit tricky - I'm no fan of the NullPointerException, but if the status really is null, that indicates a serious problem - one worthy of a RuntimeException (since this is a sample). Swallowing this error (since there's no "else") is therefore not a good idea, but adding more logic to handle this case increases the complexity, which is also not something we really want in a sample.